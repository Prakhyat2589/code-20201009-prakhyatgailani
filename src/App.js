import React, { Component } from "react";
import './App.css';
import CarDetails from './components/cardetails';

class App extends Component {
  render() {
    return (
      <div className="App">
        <CarDetails/>
      </div>
    );
  }
}
export default App;
