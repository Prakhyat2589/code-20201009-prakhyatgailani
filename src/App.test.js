import { render } from "@testing-library/react";
import App from "./App";

describe("Unit test for App", () => {
  test("renders App", async () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
