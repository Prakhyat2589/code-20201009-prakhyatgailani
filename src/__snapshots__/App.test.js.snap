// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit test for App renders App 1`] = `
<div>
  <div
    class="App"
  >
    <h1>
      Car Listings
    </h1>
    <div
      class="sc-bdfBQB bSCONV"
    >
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX sc-hKgJUU fNPEMh ifIJfe"
        >
          CAR ID
        </div>
        <div
          class="sc-dlfnuX sc-hKgJUU fNPEMh ifIJfe"
        >
          CAR NAME
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          440
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          ASTON MARTIN
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          441
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          TESLA
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          442
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          JAGUAR
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          443
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          MASERATI
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          444
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          LAND ROVER
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          445
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          ROLLS ROYCE
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          446
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          BUELL (EBR)
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          447
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          JIALING
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          448
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          TOYOTA
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          449
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          MERCEDES-BENZ
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          450
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          FREIGHTLINER
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          451
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          FULMER FABRICATIONS
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          452
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          BMW
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          453
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          MOTORCYCLE MACHINE SHOP SERVICES
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          454
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          BUGATTI
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          455
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          HOMBILT TRAILERS
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          456
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          MINI
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          457
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          ACG
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          458
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          MOONLIGHT CHOPPERS
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          459
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          COOPER CYCLES
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          460
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          FORD
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          461
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          LATITUDE
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          462
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          COUNTRY COACH
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          463
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          WESTWARD INDUSTRIES
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          464
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          LINCOLN
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          465
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          MERCURY
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          466
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          LOTUS
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          467
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          CHEVROLET
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          468
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          BUICK
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          469
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          CADILLAC
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          470
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          HOLDEN
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          471
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          OPEL
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          472
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          GMC
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          473
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          MAZDA
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          474
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          HONDA
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          475
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          ACURA
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          476
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          DODGE
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          477
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          CHRYSLER
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          478
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          NISSAN
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          479
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          QUEST MANUFACTURING
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          480
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          INFINITI
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          481
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          MITSUBISHI
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          482
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          VOLKSWAGEN
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          483
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          JEEP
        </div>
      </div>
      <div
        class="sc-gsTEea hFEVSK"
      >
        <div
          class="sc-dlfnuX fNPEMh"
        >
          484
        </div>
        <div
          class="sc-dlfnuX fNPEMh"
        >
          CLUB CAR
        </div>
      </div>
    </div>
  </div>
</div>
`;
